### PR TITLE
Set up the Sisense Connector package structure

### PR DESCRIPTION
**Pull request by ricardolsmendes**
**- What I did**
Set up the Sisense Connector package structure. 

**- How I did it**
1. Created a Python package with unit test support.
2. Created a `docs` folder with the initial developer resources.
3. Added a Postman collection to demonstrate foundational REST operations such as authentication, get folders, get dashboards and get widgets.

**- How to verify it**
Check the static content, the Postman files, and run the unit tests.

**- Description for the changelog**
Set up the Sisense Connector package structure.

PS: This PR is part of the effort to implement #70.

### CODE REVIEW

**Comment by ricardolsmendes** [comment]
Hey @mesmacosta, could you take a look, please? Thanks!

File: google-datacatalog-sisense-connector/setup.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import setuptools
+
+release_status = 'Development Status :: 4 - Beta'
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+setuptools.setup(
+    name='google-datacatalog-sisense-connector',
+    version='0.1.0',
```
Comment (mesmacosta) [inline google-datacatalog-sisense-connector/setup.py:26]: Any reason to start at `0.1.0`?

File: google-datacatalog-sisense-connector/docs/developer-resources/sisense-apis.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Sisense APIs
+
+The below API docs were used as references to develop this connector:
+
+1. [Sisense REST API v1.0 Reference (Sisense V6 and
+   higher)](https://sisense.dev/reference/rest/v1.html)
```
Comment (mesmacosta) [inline google-datacatalog-sisense-connector/docs/developer-resources/sisense-apis.md:6]: Any reason you are using the v1.0 and not v2.0 (latest) of their REST API? 
https://sisense.dev/reference/rest/v2/?platform=linux&spec=L2021.3.0

**Review by mesmacosta — APPROVED** [review]
LGTM! Nice small PR :)

Just added some minor questions.

File: google-datacatalog-sisense-connector/setup.py (hunk 1 above)
Comment (ricardolsmendes) [reply google-datacatalog-sisense-connector/setup.py]: Nice catch. I changed to `0.0.1` and will bump to `0.1.0` before publishing the first working version.

File: google-datacatalog-sisense-connector/docs/developer-resources/sisense-apis.md (hunk 2 above)
Comment (ricardolsmendes) [reply google-datacatalog-sisense-connector/docs/developer-resources/sisense-apis.md:6]: This is the version available in the Sandbox environment: `Sisense Windows Version: 8.2.5.11026 v1`.

**Comment by ricardolsmendes** [comment]
> LGTM! Nice small PR :)

Thanks for the quick response! I'll try to keep the next ones as succinct as I can, it will help to expedite the code review. 😃

**Pull request merged**